### ISSUE TITLE
Loader for Samsung Galazy SM-J510FN

### ISSUE THREAD

**Issue by abebeos** [bkerler/Loaders#109]
I've looked up half of the internet, could not find a loader.

Anyone knows if some other loader may be compatible? Need it mainly to dump the eMMC (device is bricked, comes up in EDL mode only)

```
PS P:\_own\samsungGalaxy> edl /l
Found EDL 9008, handshaking... version 2.1
HWID: 007050e100000000, JTAG: 007050e1, OEM: 0000, Model: 0000
Hash: fd174cbc0e0de003-83238f5776fa35cd-7cb7f64e9789c21c-18af09314fac8b78 (x3)
```

**Comment by RenateUSB** [comment]
I don't see that hash anywhere.

**Comment by abebeos** [comment]
> anywhere.

You mean within this repo? Or did you lookup within other sources, too?

**Comment by RenateUSB** [comment]
I meant **EVERYWHERE**. I am all-seeing.

No, I checked this repo, the Russian repo and the little Samsung one. 1097 total of which 917 are unique. Why? Do you know of other repos?

**Comment by abebeos** [comment]
No,. Me new to this problem-domain (though not new to electronics design). Documentation is bad, I gues those who really know keep it for their commercial products. But even there, there's no clear talk about "which phone has a loader and which one not".

I remember seeing on commercial support telling "j510 has no loader, you need to use jtag/direct access".

Do I understand right that EDL without a loader is useless (e.g. emmc can neither be read nor written) ?

(I'm preparing to got the hardware way - me just to bored to tinker jtag cables/adapter/software etc. Lady that gave me the hardbricked phone has a video on it from her fathar that died few years ago. That's the only reason I'm still insisting. Otherwise the phone is worthless.

**Comment by RenateUSB** [comment]
Unless it's a really old Android all the user data is encrypted and reading the eMMC on or off the board will do you no good. You need to make Android decrypt it.

**Comment by abebeos** [comment]
> Unless it's a really old Android

I think it's Android 6

**Comment by RenateUSB** [comment]
Well, you might find that it's not encrypted. There are breakout boards for eMMC. The pin count is not too bad. Some people have even wired them up "dead bug" style.

OTOH, I like these sockets (not an Amazon recommendation) https://www.amazon.com/ALLSOCKET-Programming-Chip-off-Extractor-Black-SD/dp/B076HQ1L5P

Unless you're starting a business, a bit hard to justify.

**Comment by abebeos** [comment]
> Unless you're starting a business, a bit hard to justify.

The data has a sentimental - priceless.

But those adapters are not needed.

Hardware stuff is peanuts for me, I'm just too bored to do it, so tried to avoid it via a software attempt.

**Comment by abebeos** [comment]
> Anyone knows if some other loader may be compatible?
